internal/configs/version1/template_helper.go

```diff
@@ -5,18 +5,17 @@ import (
 	"text/template"
 )
 
-// SplitInput splits the input from "," and returns an array of strings
-func SplitInput(s string, delim string) []string {
+// splitinput splits the input from "," and returns an array of strings
+func splitinput(s string, delim string) []string {
```

Comment:
The documentation for the function is not correct. I suggest removing it complety - the function is private and also strait-forward.
Same applies for triminput.